README.ubuntu.bash

```diff
@@ -1,42 +1,28 @@
 #!/bin/bash
 
-apt-get update
-apt-get install -y \
+sudo apt-get update
+sudo apt-get install -y \
   bison \
-  ccache \
   cmake \
   curl \
   flex \
-  git-core \
-  gcc \
   g++ \
-  inetutils-ping \
-  krb5-kdc \
-  krb5-admin-server \
+  gcc \
+  git \
   libapr1-dev \
   libbz2-dev \
-  libcurl4-gnutls-dev \
+  libcurl4-openssl-dev \
   libevent-dev \
   libkrb5-dev \
-  libpam-dev \
   libperl-dev \
   libreadline-dev \
   libssl-dev \
   libxml2-dev \
   libyaml-dev \
   libzstd-dev \
-  locales \
   net-tools \
-  ninja-build \
   openssh-client \
   openssh-server \
-  openssl \
-  python-dev \
-  python-pip \
-  python-psutil \
-  python-yaml \
+  python2 \
+  python2-dev \
```

Comment:
This list of packages seems shorter and some packages could be missing. Could you give an explanation why missing packages are not needed anymore? ccache seems to be used, etc

Reply:
Python-psutil and python-yaml are already integrated into the project as submodules, so we can remove them from the dependencies list.

Regarding the removal of ccache, inetutils-ping, locales, ninja-build, conan, openssl, krb5-kdc, krb5-admin-server, and libpam-dev, I don't have a definitive answer. However, I can confirm that on a clean install of Ubuntu 20.04 and 22.04 in my VirtualBox, these packages were not required to configure, make, install the current version of GPDB6, or to start the standard demo-cluster with mirrors on the same machine.

Reply:
Following this instruction psutils are not found unlike the previous document version. Building on a fresh ubuntu 22.04 (server, minimal) installation was successful, but starting demo cluster failed because of ImportError: No module named psutil

Reply:
Psutil is already integrated into the project as a submodule here: `/gpMgmt/bin/pythonSrc/ext/psutil-5.7.0.tar.gz`, and during compilation, it should be automatically installed here: `gpdb_bin/lib/python/psutil`. I have sometimes encountered similar issues with psutil or yaml. This should help:
- Ensure that you are using the adb-6.x branch.
- Update the submodules:
  ```bash
  git submodule update --init --recursive --force  # Updates changing modules
  ```
- Clear the source directory of old compilation files, recompile, and reinstall gpdb:
  ```bash
  make uninstall
  git clean -xdf
  git reset --hard
  ./configure --enable-debug-extensions --with-perl --with-pythonsrc-ext --with-python --with-libxml --with-gssapi --prefix=$HOME/gpdata/gpdb_bin CFLAGS="-O0 -g3"
  make -s -j`nproc`
  make -s install
  ```
If after these steps the demo cluster does not start correctly, let’s revert explicitly reinstalling psutil and yaml in instruction. Let me know please.

Reply:
Right, these instructions are necessary and form complete list which new user has to perform to get GP/ADB up and running. I'd suggest to put them into readme as well as it might be not obvious to newcomers. At least git submodule thing is needed.

Also, on my fresh ubuntu server 22.04 minimal I had to do the following to get it running:

`sudo apt-get install iputils-ping`
ping required for demo cluster

```
sudo apt-get install -y locales
sudo locale-gen "en_US.UTF-8"
```
locale en_US.UTF-8 is not here by default.

`sudo ldconfig`
it was needed to do after xerces was installed by the instruction.